pages/wishlist.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Aamir Nathani" />
+    <link rel="icon" href="../assets/icon-logo-1.jpg" />
+    <title>Wishlist - Watches World</title>
+
+    <!-- nomark-ui Import -->
+    <link
+      rel="stylesheet"
+      href="https://nomark-ui.netlify.app/styles/nomark.css"
+    />
+
+    <!-- Internal Import -->
+    <link rel="stylesheet" href="../style/wishlist.css" />
+    <link rel="stylesheet" href="../style/watch-common.css" />
+  </head>
+  <!-- navbar -->
+  <nav class="navbar-container sticky z-5">
+    <div class="navbar-logo-container">
+      <a href="../index.html"
+        ><img
+          src="../assets/watches-world-logo.png"
+          alt="watches-world-logo"
+          class="navbar-logo"
+      /></a>
+    </div>
+    <a class="open-menu font-size-6"><i class="fas fa-bars"></i></a>
+    <a class="close-menu font-size-6"><i class="fa fa-times"></i></a>
+    <div class="navbar-items-container">
+      <div class="navbar-search-container border-radius-2">
+        <input
+          type="text"
+          class="navbar-search"
+          placeholder="What you are looking for?"
+        />
+        <i class="fas fa-search navbar-search-btn"></i>
+      </div>
+      <ul class="navbar-links">
+        <li>
+          <a href="" class="border-radius-2"
+            ><i class="fas fa-user"></i>Logout</a
+          >
+        </li>
+        <li>
+          <a href="" class="border-radius-2"
```

Comment:
Format the code a bit so that it is easier to read.

Reply:
ok i will look into it